manim/scene/scene_file_writer.py

```diff
@@ -732,21 +751,17 @@ def combine_to_movie(self):
             # but tries to call ffmpeg via its CLI -- which we want
             # to avoid. This is why we need to do the conversion
             # manually.
-            if config.format == "webm":
-                with (
-                    av.open(sound_file_path) as wav_audio,
-                    av.open(sound_file_path.with_suffix(".ogg"), "w") as opus_audio,
-                ):
-                    wav_audio_stream = wav_audio.streams.audio[0]
-                    opus_audio_stream = opus_audio.add_stream("libvorbis")
-                    for frame in wav_audio.decode(wav_audio_stream):
-                        for packet in opus_audio_stream.encode(frame):
-                            opus_audio.mux(packet)
-
-                    for packet in opus_audio_stream.encode():
-                        opus_audio.mux(packet)
-
-                sound_file_path = sound_file_path.with_suffix(".ogg")
+            out_suffix = movie_file_path.suffix.lower()
+            if config.format == "webm" or out_suffix == ".webm":
```

Comment:
For webms, I am not aware of a situation where `config.format` could misalign with the constructed movie suffix. Did you have anything in particular in mind here?

Reply:
If not:

```suggestion
            if config.format == "webm":
```

Reply:
For consistency with mp4 below, mostly, but also because `config.format` may be `None`, when `--format` is not specified, in which case libav takes a guess from the file suffix, I believe. BTW the suffix is the only clue libav has, isn't it? I cannot see anything that would override the container format.